### DISCUSSION TITLE
不能用每日音乐推荐吗😭

### DISCUSSION THREAD

**Discussion by komiblog** [HyPlayer/HyPlayer#223] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/102006886/182520388-91eaf1d0-ea3c-4e4c-869c-57b6808c7a01.png)

**Comment by kengwang** [comment, marked as answer]
可以使用的
![image](https://user-images.githubusercontent.com/30862240/182521364-02430055-f4b3-44c3-9247-54dd4d7183b0.png)
请登录网页版 网易云音乐 进入个人设置 - 隐私设置 - 个性化服务 是否开启

**Reply by komiblog** [reply]
多谢解答😊
